### DISCUSSION TITLE
Unable to get access to outlook mailbox

### DISCUSSION THREAD

**Discussion by devarajan88** [microsoftgraph/msgraph-sdk-go#652] [General]
Dear Contributors,

I am using below code snippets to access mailbox,

	cred, _ := azidentity.NewClientSecretCredential(
		TENANT_ID,
		CLIENT_ID,
		CLIENT_SECRET,
		nil,
	)

	graphClient, _ := graph.NewGraphServiceClientWithCredentials(
		cred, []string{"https://graph.microsoft.com/.default"})

	result, err := graphClient.Me().Drive().Get(context.Background(), nil)

	if err != nil {
		fmt.Printf("Error getting the drive: %v\n", err)
		fmt.Println(err)
	}
	fmt.Printf("Found Drive : %v\n", *result.GetId())

But, I always get this error below,

I configured app registration as guided in the microsoft official docs.

Please guide me how to get rid of this error.

Thanks and appreciate your prompt on this issue.
Deva

**Comment by rkodev** [comment]
Hi @devarajan88 it looks like you failed to show us the error message from the server
